test/e2e/app-dir/rsc-basic.test.ts

```diff
@@ -390,40 +366,29 @@ export default function (context, { runtime, env }) {
     )
   })
 
-  // TODO: _app + next/head
-  it.skip('should support next/head inside _app with RSC', async () => {
-    const browser = await webdriver(context.appPort, '/multi')
-    const title = await browser.eval(`document.title`)
-    expect(title).toBe('hi')
-  })
-}
-
-/*
-TODO: re-enable css suite once css is supported in app-dir
-
-it.skip('should include global styles with `serverComponents: true`', async () => {
-  const browser = await webdriver(context.appPort, '/global-styles-rsc')
-  const currentColor = await browser.eval(
-    `window.getComputedStyle(document.querySelector('#red')).color`
-  )
-  expect(currentColor).toMatchInlineSnapshot(`"rgb(255, 0, 0)"`)
-})
-
-it.skip('should include css modules with `serverComponents: true`', async () => {
-  const browser = await webdriver(context.appPort, '/css-modules')
-  const currentColor = await browser.eval(
-    `window.getComputedStyle(document.querySelector('h1')).color`
-  )
-  expect(currentColor).toMatchInlineSnapshot(`"rgb(255, 0, 0)"`)
+  if (!isNextDev) {
+    it('should generate edge SSR manifests for Node.js', async () => {
+      const distServerDir = path.join(distDir, 'server')
+
+      const requiredServerFiles = (
```

Comment:
We need to wrap any checks for the manifest in a `!(global as any).isNextDeploy` since we can't read the build outputs for deploy tests. 

x-ref: https://github.com/vercel/next.js/commit/fd3a2af4aa1285291e29e06937d85d02d9b54a80

Reply:
I have disabled the deploy tests for rsc now

https://github.com/vercel/next.js/pull/38460/files#diff-c351cb3217d38df3c5b463578fe5b809e72b7a70748f64ae5bca46fdc66766a8R52-R55